content/doc/developer/security/csp.adoc

```diff
@@ -27,6 +27,8 @@ Inline `<script>` blocks::
 Referencing a file to load JavaScript from is fine, but inline block contents are not.
 All `<script>` tags without `src` are a problem.
 They will break without `'unsafe-inline'` in the `script-src` CSP directive.
+Standalone `st:bind` tags with disallowed `var` attribute::
```

Comment:
Or with no `var` attribute at all? That would likely mean it's used inside a script block, so unsure if it qualifies.

Reply:
Then it's not "standalone" (which I use as a term referring to the non-inline use).